src/Notifynder/Traits/NotifableBasic.php

```diff
@@ -135,4 +135,22 @@ public function getNotifications($limit = null, $order = 'desc')
 
         return $query->get();
     }
+
+    /**
+     * Get all unread Notifications.
+     * 
+     * @param null|int $limit
+     * @param string $order
+     * @return \Illuminate\Database\Eloquent\Collection
+     */
+    public function getNotificationsNotRead($limit = null, $order = 'desc')
+    {
+        $query = $this->getNotificationRelation()->orderBy('created_at', $order);
+        $query = $query->where('notifications.read', 0);
```

Comment:
We don't use the table name in queries cause this can be changed - so pls just `where('read', 0)`